src/graph/sampling/randomwalks/node2vec_randomwalk.h

```diff
@@ -163,7 +163,7 @@ std::pair<IdArray, IdArray> Node2vecRandomWalk(
                                                   edges, csr_has_data, prob, terminate);
     };
 
-  return GenericRandomWalk<XPU, IdxType>(seeds, max_num_steps, step);
+  return GenericRandomWalk<XPU, IdxType>(seeds, max_num_steps, step, g->NumVertices(0));
```

Comment:
Is this always etype 0?

Reply:
Yes.  Node2vec currently only supports homogeneous graphs.